docs-source/spring/data/menu/main.yaml

```diff
@@ -1,6 +1,11 @@
 main:
   - name: Setup
     ref: "/setup"
+    sub:
```

Comment:
do not move these. put them back where they were so that 1) it is consistent with mbaas docs, and 2) database is not hidden away in the weeds